# these are just the two angles for the pan/tilt rig:
float32 joint1
float32 joint2
float32 joint3
float32 joint4
float32 joint5
float32 joint6
